test/EndToEnd/tests/LegacyPackageRefProjectTest.ps1

```diff
@@ -26,4 +26,54 @@ function Test-UwpPackageRefClassLibInstallPackage {
     Assert-AreEqual 2 $packageRefs.Count
     Assert-AreEqual $packageRefs[1].GetMetadataValue("Identity") 'Nuget.Versioning' 
     Assert-AreEqual $packageRefs[1].GetMetadataValue("Version") '3.5.0'
+}
+
+function Test-NetCorePackageVersionNoInclusiveLowerBoundNU1604 {
+
+    # Arrange
+    param(
+        $context
+    )    
+
+    $sol = New-Solution
+    $solutionDir = Get-SolutionDir
+
+    # Create Directory.Build.props file with PackageReference with invalid version.
+    $directoryPropPath = Join-Path $solutionDir "Directory.Build.props"
+	$directoryPropContent =@"
+<Project>
+    <ItemGroup>
+      <PackageReference Include="TestUpdatePackage"/>
+    </ItemGroup>
+  
+  </Project>  
+"@   
+
+    $directoryPropContent | Out-File  $directoryPropPath   
+
+    $project = $sol | New-WpfApplication
+
+    # Act
+    Build-Solution
+
+    # Assert   
+    # Make sure VS UI warning has NU1604 warning       
+    Start-Sleep -s 3 # Give time for restore to finish, so we can capture VS UI warnings.
```

Comment:
This should not be necessary. 

Build Solution runs restore and build and they are synchronous.

Reply:
Removed. It looks my problem was due to debug mode.